snap/info.go

```diff
@@ -231,6 +232,18 @@ type DownloadInfo struct {
 	Sha3_384 string `json:"sha3-384,omitempty"`
 }
 
+// DeltaInfo contains the information to download a delta
+// from one revision to another.
+type DeltaInfo struct {
+	FromRevision    int    `json:"from-revision,omitempty"`
+	ToRevision      int    `json:"to-revision,omitempty"`
```

Comment:
I think we do need both. As per the example in the test that I added, the server may respond with two (non-arbitrary) deltas to get you from 24->26 (24->25 and 25->26)... ah, and perhaps you mean that the client could still infer that with just the ToRevision. Yes, we could just use ToRevision. I'll play with that a bit more in my mind, but probably update. Thanks.